styles(notistack): made compatible with Theme (#1850)
pankod/refine: .changeset/chilled-steaks-cheat.md

---
"@pankod/refine-mui": patch
---

> Notistack SnackbarProvider which is used as a notification provider ( from `@pankod/refine-mui` ) has been made compatible with the theme in the example of the fine food.

Reviewer: ```suggestion
[Notistack](https://github.com/iamhosseindhv/notistack) `SnackbarProvider` which is used as a notification provider (from `@pankod/refine-mui`) has been made compatible with the theme in the example of the fine food.
```

> ---
> "mui-fine-foods": minor
> "@pankod/refine-mui": patch
> ---

Reviewer: ```suggestion
---
"@pankod/refine-mui": patch
---
```
We don't need examples changelogs

> Notistack SnackbarProvider which is used as a notification provider ( from @pankod/refine-mui ) has been mmade compatible with the theme in the fine foods example.

Reviewer: ```suggestion
Notistack SnackbarProvider which is used as a notification provider ( from `@pankod/refine-mui` ) has been made compatible with the theme in the example of the fine food.
```
